plugins/task-resources/src/components/kanban/KanbanView.svelte

```diff
@@ -87,7 +88,7 @@
   }
 
   let resultQuery: DocumentQuery<any> = { ...query }
-  $: getResultQuery(query, viewOptionsConfig, viewOptions).then((p) => (resultQuery = { ...p, ...query }))
+  $: getResultQuery(query, viewOptionsConfig, viewOptions).then((p) => (resultQuery = mergeQueries(p, query)))
```

Comment:
We should reverse order here?
```suggestion
  $: getResultQuery(query, viewOptionsConfig, viewOptions).then((p) => (resultQuery = mergeQueries(query, p)))
```

So we will merge into base one, filter ones.?

Reply:
It should not care about the order of arguments, so supposed to give the same answer no matter of the order